airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcSqlGenerator.kt

```diff
@@ -285,25 +292,31 @@ abstract class JdbcSqlGenerator(protected val namingTransformer: NamingConventio
     }
 
     override fun updateTable(
-        streamConfig: StreamConfig,
+        stream: StreamConfig,
         finalSuffix: String,
         minRawTimestamp: Optional<Instant>,
         useExpensiveSaferCasting: Boolean
     ): Sql {
         // TODO: Add flag to use merge vs insert/delete
 
         return insertAndDeleteTransaction(
-            streamConfig,
+            stream,
             finalSuffix,
             minRawTimestamp,
             useExpensiveSaferCasting
         )
     }
 
     override fun overwriteFinalTable(stream: StreamId, finalSuffix: String): Sql {
+        val dropTableStep = DSL.dropTableIfExists(DSL.name(stream.finalNamespace, stream.finalName))
+        val dropTableSql =
+            if (cascadeDrop) {
+                dropTableStep.cascade().getSQL(ParamType.INLINED)
+            } else {
+                dropTableStep.getSQL(ParamType.INLINED)
+            }
```

Comment:
kotlin expression ? 
```
dropTableSql = if (cascade) .. else ..
```

Reply:
it's already an expression, the autoformatter just moved it to a new line. I'm happy to remove the braces, no strong format preference about that. autoformatter produces this:
```kotlin
        val dropTableSql =
            if (cascadeDrop) dropTableStep.cascade().getSQL(ParamType.INLINED)
            else dropTableStep.getSQL(ParamType.INLINED)
```